### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by craigs40**
In this project, we will refactor the Awesome books application to use JavaScript classes.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+/* Add Book */
 const addBook = () => {
   let books;
   const bookTitle = document.querySelector('#title').value;
```
Comment (Lameck1) [inline index.js]: - [x] According to this requirement 👇🏿:

  > - Create class methods to add and remove books.

  The methods for adding and removing books should be class methods. So, you ought to have refactored them into the class. Please ensure the `addBook` method is a class method.

File: index.js [hunk 2]
```diff
@@ -36,14 +45,7 @@ const displayBook = () => {
...
-  addBook();
-  displayBook();
-  document.querySelector('#title').value = '';
-  document.querySelector('#author').value = '';
-});
-
+/* Remove Book */
 const deleteBook = (elem) => {
   const books = JSON.parse(localStorage.getItem('books'));
   if (books.length === 1) {
```
Comment (Lameck1) [inline index.js]: - [x] Same issue as above. Please ensure that the `deleteBook` method is moved into the class to make it a class method.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @craigs40 & @ForHemer, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
**Hi @team,** 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**